osf/metadata/osf_gathering.py

```diff
@@ -921,6 +922,14 @@ def gather_qualified_attributions(focus):
                 yield (_attribution_ref, DCAT.hadRole, _osfrole_ref)
                 yield (_attribution_ref, OSF.order, index)
 
+@gather.er(OSF.verifiedLink)
+def gather_verified_link(focus):
+    links = focus.dbmodel.get_links()
+    for link in links:
+        ref = rdflib.BNode()
+        yield (OSF.verifiedLink, ref)
+        yield (ref, DCAT.accessURL, link['target_url'])
+        yield (ref, DATACITE.resourceTypeGeneral, link['resource_type'])
```

Comment:
Looks good to me. Please double check with @aaxelb to make sure he is good with this too. And once it's merged, I will let PO know of this so they and update their OSF Map sheet.